helpers/pluck.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
+const helpers = require('handlebars-helpers');
+
 const factory = () => {
     return function(collection, path) {
-        return collection.map(item => item[path]);
+        if (collection === null || typeof collection === 'undefined' || typeof collection !== 'object') {
```

Comment:
Is `pluck` supposed to operate on arrays or objects?

Reply:
As per our documentation both